exp/operator/api/v1alpha4/bootstrapprovider_types.go

```diff
@@ -20,19 +20,19 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// BootstrapProviderSpec defines the desired state of BootstrapProvider
+// BootstrapProviderSpec defines the desired state of BootstrapProvider.
```

Comment:
If I look at the proposal there is no `BootstrapProviderSpec`, but we are using `ProviderSpec` directly inside the the `BootstrapProvider`.
Is there a reason why you took a different approach here?

Reply:
This applies to other providers type as well

Reply:
There is no reason, it came from the initial kubebuilder scaffolding. Thank you for noticing it, I'll fix it.